Makefile

```diff
@@ -64,6 +64,7 @@ check: python-env
 	${FIND} -name *.py -exec autopep8 -d --max-line-length 120  {} \; | (! grep . -q) || (echo "Code differs from autopep8's style" && false)
 	# Validate that all updates were committed
 	$(MAKE) update
+	$(MAKE) notice
```

Comment:
Should we also add `$(MAKE) notice` to the beats `$(MAKE) update`? So it will also be run with the update command.